Flow.Launcher/ResultListBox.xaml

```diff
@@ -65,23 +65,24 @@
                             Margin="0 0 10 0"
                             VerticalAlignment="Center"
                             Visibility="{Binding ShowOpenResultHotkey}">
-                            <TextBlock
-                                x:Name="Hotkey"
-                                Margin="12 0 12 0"
-                                Padding="0 0 0 0"
-                                HorizontalAlignment="Right"
-                                VerticalAlignment="Center"
-                                Style="{DynamicResource ItemHotkeyStyle}">
-                                <TextBlock.Visibility>
+                            <Border x:Name="HotkeyBG" Style="{DynamicResource ItemHotkeyBGStyle}">
```

Comment:
do we need the Name for Border (I don't think even accessibility people need to care about the border)